### ISSUE TITLE
open code

### ISSUE THREAD

**Issue by 779257747** [zhiyuanpeng/KDD2023-EAMT#2]
where is your open code?
When will the code be released?

**Comment by zhiyuanpeng** [comment]
Yes, we do have a plan to release the code. The request to release the code was submitted for the approval from VP in Mar. However, the VP retired, and I have to ask my previous manager to submit a new ticket.
